app/oauth2-server-config/client/admin/views/oauthApp.js

```diff
@@ -11,6 +11,23 @@ import { modal, SideNav } from '../../../../ui-utils/client';
 import { t, handleError } from '../../../../utils';
 import { APIClient } from '../../../../utils/client';
 
+function isValidURL(string) {
```

Comment:
use `validateUrl` you can find here `markdown/lib/parser/original/markdown.js` please move this helper to `ui-utils/client/lib` and use in both places